about.html

```diff
@@ -51,69 +51,89 @@ <h1 class="margin-bottom large-title">About</h1>
       <div class="margin-bottom">
         <h2 class="bold">About AdoptOpenJDK - GetOpenJDK</h2>
         <p>
-          This initiative came about following a discussion about the general lack of an open and reproducible build & test system for the OpenJDK source
-          across multiple platforms.  Practically, it's a reboot of the work that has already been done for a while in the OpenJDK project producing binaries
-          for developers and end users.
+          This initiative came about following a discussion about the general lack of an open and reproducible build & test system
+          for the OpenJDK source across multiple platforms.
         </p>
 
         <p>
           The intention is to provide binaries and hardware access for all of the major platforms, for the purposes of:
           <ul>
-            <li>Research and Development by academics, researchers and enthusiasts.</li>
+            <li>Allowing end users to have a reliable source of OpenJDK binaries for all platforms.</li>
+            <li>Research and Development by OpenJDK developers, academics and researchers.</li>
             <li>An open, common, audited, build infrastructure for vendors to use (if they wish).</li>
-            <li>Providing binaries for bleeding edge early adopters.</li>
-            <li>Providing a place to try out build infrastructure ideas that might one day be moved to OpenJDK proper.</li>
+            <li>Providing a place to try out build infrastructure ideas that might one day be re-implemented in OpenJDK proper.</li>
           </ul>
         </p>
+      </div>
 
+      <div class="margin-bottom">
+        <h2 class="bold">I'm a Java / OpenJDK User</h2>
         <p>
-          Although we will be making the binaries to the highest quality possible via jtreg, jcov test coverage and possibly other donated test
-          suites. It is <strong>not</strong> the intention that the binaries produced will be professionally supported. See
-          <a href="./support.html">Support</a> for details.
+          We will be providing solidly made binaries to the highest quality possible utilising existing OpenJDK test suites
```

Comment:
Consider using present tense throughout, e.g. "We will be providing" becomes "We provide".